src/v/serde/protobuf/tests/parser_test.cc

```diff
@@ -232,8 +241,10 @@ class ProtobufParserFixture : public testing::Test {
               reflect->SetString(output, field, std::move(str));
           },
           [=, this](const parsed::map& v) {
```

Comment:
Before we never hit this case and always represented stuff as protobuf's internal representation of maps (which are synthetic repeated messages with a key and a value). Good catch @mmaslankaprv